log/log.go

```diff
@@ -39,10 +41,64 @@ func (f levelFlag) Set(level string) error {
 	return nil
 }
 
+func setSyslogFormatter(appname, facility string) error {
+	if appname != "" && facility != "" {
```

Comment:
Wait, this should have been changed to be the other way around. `appname == "" || facility == ""`.